src/components/ObsDetails/DataQualityAssessment.js

```diff
@@ -169,7 +171,7 @@ const DataQualityAssessment = ( {
           </View>
           <DQAVoteButtons
             metric="date"
-            qualityMetrics={qualityMetrics}
```

Comment:
@angielt I am only passing down the array of votes that are relevant for the particular metric. That makes the internal logic of each button more performant because it can return early if no data, or doesn't have to use Array.find/filter on a lot of elements. I have changed the qualityMetrics object so that each metric is a key that has the votes for that metric, so that data slicing should be pretty fast for this.

Reply:
Makes sense. Definitely a lot better than having each button do that find its respective metric.